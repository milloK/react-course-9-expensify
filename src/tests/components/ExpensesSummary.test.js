import React from 'react'
import { shallow } from 'enzyme'
import { ExpensesSummary } from '../../components/ExpensesSummary'

it('Should correctly render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>)
  expect(wrapper).toMatchSnapshot()
})

  it('Should correctly render ExpensesSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={2365445}/>)
  expect(wrapper).toMatchSnapshot()
})