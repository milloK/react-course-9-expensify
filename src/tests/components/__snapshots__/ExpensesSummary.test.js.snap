// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     tottaling 
    $2.35
  </h1>
</div>
`;

exports[`Should correctly render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     tottaling 
    $23,654.45
  </h1>
</div>
`;
